source/core/slang-signal.cpp

```
@@ -86,8 +86,16 @@ void handleAssert(char const* message, bool isReleaseAssert)
     }
 
-    // Default behavior: delegate to handleSignal
-    handleSignal(SignalType::AssertFailure, message);
+    // Use only the basename to avoid leaking build-machine paths in release builds.
+    const char* basename = file;
+    for (const char* p = file; *p; ++p)
+    {
+        if (*p == '/' || *p == '\\')
+            basename = p + 1;
+    }
+    StringBuilder locMsg;
```

Cppcheck (2.20.0)

[style] The function 'FastFloor' is never used.

(unusedFunction)